src/unix/bsd-proctitle.c

```diff
@@ -38,9 +38,9 @@ static void init_process_title_mutex_once(void) {
 
 
 void uv__process_title_cleanup(void) {
-  /* TODO(bnoordhuis) uv_mutex_destroy(&process_title_mutex)
-   * and reset process_title_mutex_once?
-   */
+  static const uv_once_t uv_once_init = UV_ONCE_INIT;
+  uv_mutex_destroy(&process_title_mutex);
+  memcpy(&process_title_mutex_once, &uv_once_init, sizeof(uv_once_init));
```

Comment:
This constant doesn't exist. But since the library is about to be unloaded (free'd), it does not really matter.
```suggestion
```

Reply:
EDIT: ah, it is created just above. But still, the documentation says this is undefined behavior (to reinitialize or copy them), so I am somewhat disinclined to do that.

Reply:
@vtjnash The `DESCRIPTION` section at https://pubs.opengroup.org/onlinepubs/009695299/functions/pthread_once.html says:

> The behavior of `pthread_once()` is undefined if `once_control` has automatic storage duration or is not initialized by `PTHREAD_ONCE_INIT`.

but `uv_once_init` has static storage and it is initialized to `PTHREAD_ONCE_INIT`. How is this undefined behaviour then?

Reply:
The memcpy may have undefined behavior, as it is not equivalent to "initialized by `PTHREAD_ONCE_INIT`"

Reply:
@vtjnash thanks, updated. PTAL!